app/ players/player.py

```diff
@@ -0,0 +1,16 @@
+from abc import ABC, abstractmethod

```

Comment:
The import statement should be on a separate line for each module. You should separate the imports of `ABC` and `abstractmethod` from `abc` into two lines for better readability and to follow Python's PEP 8 style guide.